api/routes/api.php

```diff
@@ -78,6 +78,13 @@
     Route::get('/emails', 'EventController@getEventsEmails');
 });
 
+Route::group([
+                 'namespace' => 'Api\\',
+                 'prefix' => '/tags'
+             ], function () {
+    Route::get('/', 'TagController@getTagsByEventTypeId');
```

Comment:
what method `getTagsByEventTypeId` do?

Reply:
It depends on how we are gonna use it. If we need we need event type tags - `/event-types/{eventTypeId}/tags`, if we need all tags - `/tags`.